packages/inventory-compliance/src/ComplianceRemediationButton/ComplianceRemediationButton.js

```diff
@@ -7,8 +7,8 @@ import { addNotification } from '@redhat-cloud-services/frontend-components-noti
 import { AnsibeTowerIcon } from '@patternfly/react-icons';
 
 class ComplianceRemediationButton extends React.Component {
-    formatRule = ({ title, refId }, profile, system) => ({
-        id: `ssg:rhel7|${profile.split('xccdf_org.ssgproject.')[1]}|${refId}`,
+    formatRule = ({ title, refId }, profile, system, majorOsVersion) => ({
+        id: `ssg:rhel${majorOsVersion}|${profile.split('xccdf_org.ssgproject.')[1]}|${refId}`,
```

Comment:
I think we need some validation here to ensure `majorOsVersion` is not `undefined`.

![image](https://user-images.githubusercontent.com/761923/109181060-85789900-7759-11eb-9761-1ee8a7ff8a18.png)

Reply:
I opened a PR to request the OS version on the report details as well. It shouldn't be undefined anymore with that for this case: https://github.com/RedHatInsights/compliance-frontend/pull/1019

We could set a default in the buttons `formatRules` function, but the user would still get a similar error.